ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/config/CCDAdapterConfig.java

```diff
@@ -103,7 +107,10 @@ public ObjectMapper ccdObjectMapper() {
             .addMixIn(ClaimData.class, ClaimDataMixIn.class)
             .addMixIn(ContactDetails.class, ContactDetailsMixIn.class)
             .addMixIn(Claim.class, ClaimMixIn.class)
-            .addMixIn(Representative.class, RepresentativeMixIn.class);
+            .addMixIn(Representative.class, RepresentativeMixIn.class)
+            .addMixIn(ClaimantResponse.class, ClaimantResponseMixin.class)
+            .addMixIn(PaymentIntention.class, PaymentIntentionMixin.class)
+            .enable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
this will enable pretty printing, but the output is going to be sent as JSOn payload, do we need it?